features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/h2.sql

```diff
@@ -1326,12 +1326,34 @@ CREATE TABLE IF NOT EXISTS SCOPE (
     API_ID VARCHAR(255) NOT NULL,
     NAME VARCHAR(255) NOT NULL,
     DISPLAY_NAME VARCHAR(255) NOT NULL,
-    TENANT_ID INT NOT NULL,
     DESCRIPTION VARCHAR(300),
+    TENANT_ID INT NOT NULL,
```

Comment:
why the column order has been changed in this script?

Reply:
revereted.